llvm/test/Transforms/LoopUnroll/unroll-remove-redundant-dbg.ll

```diff
@@ -0,0 +1,51 @@
+; RUN: opt < %s -S -mtriple=riscv64 -passes=loop-unroll | FileCheck %s
```

Comment:
Thanks for the patch! Does the test require `-mtriple=riscv64` / `target triple = "riscv64"`? If not, please remove the references to RISCV. Otherwise it needs to be moved to `llvm/test/Transforms/LoopUnroll/RISCV/`, as it will fail on systems that don't build the RISCV backend.